File: PYTHON/Numeric-Functions/absolute_value.py
def absolute_value (num):
	num = int(num)
	if (num >= 0):
		return num
	else:
		return -num

x = int(input("User, enter a number: "))
results = absolute_value(x)
print("|" + str(x) + "| --> " + str(results))

File: C++/code/absolute_value.cpp
# include <iostream>
using namespace std;

int absolute_value (int num) {
	if (num >= 0) {
		return num;
	}
	else {
		return -num;
	}
}

int main () {
	int x;
	int results;
	cout << "User, enter a number: ";
	cin >> x;
	results = absolute_value(x);
	cout << "|" << x << "| --> " << results << "\n";
}
